x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/services/elastic/action/ElasticInferenceServiceActionCreator.java

```diff
@@ -43,4 +45,13 @@ public ExecutableAction create(ElasticInferenceServiceSparseEmbeddingsModel mode
         );
         return new SenderExecutableAction(sender, requestManager, errorMessage);
     }
+
+    @Override
+    public ExecutableAction create(ElasticInferenceServiceRerankModel model) {
+        var requestManager = new ElasticInferenceServiceRerankRequestManager(model, serviceComponents, traceContext);
+        var errorMessage = constructFailedToSendRequestMessage(
+            String.format(Locale.ROOT, "%s rerank", ELASTIC_INFERENCE_SERVICE_IDENTIFIER)
```

Comment:
nit: I think `Strings.format()` avoids the need for `Locale.ROOT`.

Reply:
[Use Strings.format(...)](https://github.com/elastic/elasticsearch/pull/126022/commits/f43417b6d8bd2fe70e1b98d9dafddf85d7f5b4c0)

Reply:
Ah, adjusted it for sparse embeddings: [Use Strings.format(...) for rerank action, too](https://github.com/elastic/elasticsearch/pull/126022/commits/c0d8a300de210534ca845e50bf7c198f204330ce)